adobe_vipm/adobe/config.py

```diff
@@ -14,13 +15,16 @@
     CountryNotFoundError,
     ResellerNotFoundError,
 )
-from adobe_vipm.utils import find_first
+from adobe_vipm.utils import find_first, map_by
+
+REQUIRED_API_SCOPES = ("openid", "AdobeID", "read_organizations")
 
 
 class Config:
-    REQUIRED_API_SCOPES = ["openid", "AdobeID", "read_organizations"]
+    """Adobe Configuration."""
 
     def __init__(self) -> None:
+        """Initialize configuration."""
```

Comment:
After add [D107](https://github.com/softwareone-platform/mpt-api-python-client/pull/4), I believe we can remove this docstrings. It doesn't give any relevant information.